jina/docker/hubio.py

```diff
@@ -73,7 +74,46 @@ def new(self):
             self.logger.info('nothing is created, bye!')
 
     def push(self, name: str = None, readme_path: str = None):
-        """A wrapper of docker push """
+        """ A wrapper of docker push """
+        is_build_success, is_push_success = True, False
+        try:
+            # self._push(name, readme_path)
```

Comment:
Do we intend on calling `_push` from here? It could return `True` on successful push and could be used for setting `is_push_success`
It could be renamed differently for more clarity.

Could you document what the args `name` and `readme_path` refer to?

Reply:
Yes. That was an unintended comment. Fixed the pytest failure too.